### PR TITLE
ManageIQ Appliance install on IBM Cloud

### PR DESCRIPTION
**Pull request by dehawkins512**
@Fryguy This is not finalized yet, but I want to get this in for baseline.  I need to see how the images look, formatting, etc.

### CODE REVIEW

**Comment by dehawkins512** [comment]
https://github.com/ManageIQ/manageiq-documentation/issues/1473

File: installing_on_ibm_cloud/_topics/installation.md [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+
+1. If you don't already have an instance of IBM Cloud Object Storage, see [Getting started with IBM Cloud Object Storage](https://cloud.ibm.com/docs/services/cloud-object-storage?topic=cloud-object-storage-getting-started)
+
+    ![Figure showing example Cloud Object Storage created.](../images/cloud_object_storage.png){: caption="Figure 2. Example Cloud Object Storage created" caption-side="bottom"}
+
+    You must also create a bucket in IBM Cloud Object Storage to store your images.
+    ![Figure showing example standard type bucket created.](../images/buckets.png){: caption="Figure 3. Example Standard type bucket created" caption-side="bottom"}
+
+
+2. Upload the {{ site.data.product.title }} installation image (file name: `cfme-rhos-5.11.4.x86_64.qcow2`) to your IBM Cloud Object Storage. Select your bucket and click Add Objects to upload the images. For more information, see [Uploading data by using the console](https://cloud.ibm.com/docs/services/cloud-object-storage?topic=cloud-object-storage-upload#upload-console). **Note:** You can use the Aspera high-speed transfer plug-in to upload images larger than 200 MB.
```
Comment (Fryguy) [inline installing_on_ibm_cloud/_topics/installation.md:39]: Should the image name be whatever the upstream name is?

Comment (dehawkins512) [reply installing_on_ibm_cloud/_topics/installation.md:39]: Good catch. I will update the name AND the image displayed so they both reflect the .q2 file.  stand by.

**Comment by dehawkins512** [comment]
@Fryguy Updated the filename to match the .q2 filename downloaded from https://www.manageiq.org/download/

**Comment by dehawkins512** [comment]
@fryguy This PR is ready for final review / merge.

**Comment by chessbyte** [comment]
@dehawkins512 if the PR is ready for final review/merge, maybe you should remove the word `Draft` from the title of this PR.

**Pull request merged**